leetcode/string.md

```diff
@@ -0,0 +1,22 @@
+# Intervals
```

Comment:
Your pull request: LEETCODE-1022 Added reverse-words-in-a-string-iii


Your commit: LEETCODE-1022 Added reverse-words-in-a-string-iii

Your commit: LEETCODE 1022 Refactored reverse-words-in-a-string
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Number of your group must be from ['1021', '1022']
Your commit: LEETCODE-1022 Refactored reverse-words-in-a-string-iii
Action of your comment must be from ['Added', 'Deleted']
Your commit: Merge remote-tracking branch 'origin/reverse-words-in-a-string-iii' into reverse-words-in-a-string-iii

# Conflicts:
#	leetcode/string.md
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Number of your group must be from ['1021', '1022']
Action of your comment must be from ['Added', 'Deleted']
Your commit: LEETCODE-1022 Refactored reverse-words-in-a-string-iii
Action of your comment must be from ['Added', 'Deleted']
Your commit: LEETCODE-1022 Refactored reverse-words-in-a-string-iii
Action of your comment must be from ['Added', 'Deleted']